File: apple_and_orange.py
def countApplesAndOranges(s, t, a, b, apples, oranges):
    apple_yield = 0
    orange_yield = 0
    for apple in apples:
        if a + apple >= s and a + apple <= t:
            apple_yield +=1
    for orange in oranges:
        if b + orange >= s and b + orange <= t:
            orange_yield += 1
    print(f"{apple_yield}\n{orange_yield}")

File: appleAndOrange.js
function countApplesAndOranges(s, t, a, b, apples, oranges){
  let appleYield = 0
  let orangeYield = 0
  for(let apple of apples) {
    if(a + apple >= s && a + apple <=t) {
      appleYield += 1
    }
  }
  for(let orange of oranges) {
    if(b + orange >= s && b + orange <=t) {
      orangeYield += 1
    }
  }
  console.log(`appleYield
  orangeYield`)
}
